README.md

```diff
@@ -1,116 +1,20 @@
 # ERC-5792 Extension for Scaffold-ETH 2
 
-This extension demonstrates on how to use [EIP-5792](https://eips.ethereum.org/EIPS/eip-5792) wallet capabilities. This EIP introduces new JSON-RPC methods for communication between apps and wallets. Which allows more advanced interactions like submitting multiple onchain calls as part of a single
-transaction or sponsoring users transactions via ERC-4337 paymasters.
+This extension demonstrates an example on how to ask a [EIP-5792](https://eips.ethereum.org/EIPS/eip-5792) compliant wallet to process a batch of onchain write calls and to check on the status of those calls.
 
-This extension comes with an example frontend interaction with the `EIP5792_Example.sol` contract, the code demonstrates on how to make an batched transaction that sets new greetings and increments the counter in a single transaction using wagmi's experimental hooks and coinbase smart wallet.
+This extension comes with an example of frontend interaction (`packages/nextjs/app/eip-5792/_components/EIp5792Example.tsx`) with the `EIP5792_Example.sol` contract, the code demonstrates on how to make batched calls to the burner wallet (on local chain) or any eip-5792 [compliant wallet](https://www.eip5792.xyz/ecosystem/wallets), that batches new greetings and increments the counter call in a single call to wallet using wagmi's hooks.
```

Comment:
metamask is in compliant wallets list but it doesnt work for me. Coinbase wallet working great

Reply:
Ohh probably it will become compliant when you switch the account to smart account in metamask: 

<img width="391" alt="image" src="https://github.com/user-attachments/assets/b2747d7a-cc51-4483-b08e-834054c2d999" />